ci-operator/templates/openshift/openshift-azure/cluster-launch-e2e-azure-bushslicer.yaml

```diff
@@ -0,0 +1,245 @@
+kind: Template
+apiVersion: template.openshift.io/v1
+
+parameters:
+- name: JOB_NAME_SAFE
+  required: true
+- name: JOB_NAME_HASH
+  required: true
+- name: LOCAL_IMAGE_BIN
+  required: true
+- name: IMAGE_SYNC
+  required: true
+- name: IMAGE_LOGBRIDGE
+  required: true
+- name: NAMESPACE
+  required: true
+- name: CLUSTER_TYPE
+  value: "azure"
+  required: true
+- name: TEST_IN_PRODUCTION
+- name: DEPLOY_VERSION
+  value: "v3.11"
+
+objects:
+
+# We want the cluster to be able to access these images
+- kind: RoleBinding
+  apiVersion: authorization.openshift.io/v1
+  metadata:
+    name: ${JOB_NAME_SAFE}-image-puller
+    namespace: ${NAMESPACE}
+  roleRef:
+    name: system:image-puller
+  subjects:
+  - kind: SystemGroup
+    name: system:unauthenticated
+
+- kind: Pod
+  apiVersion: v1
+  metadata:
+    name: ${JOB_NAME_SAFE}
+    namespace: ${NAMESPACE}
+    annotations:
+      # we want to gather the teardown logs no matter what
+      ci-operator.openshift.io/wait-for-container-artifacts: teardown,test
+      ci-operator.openshift.io/containers-logged-on-failure: setup,test,teardown
```

Comment:
This is a different annotation now. `ci-operator.openshift.io/save-container-logs: "true"`